trip_journal_app/static/javascript/login_by_mail.js

```diff
@@ -0,0 +1,127 @@
+window.onload = function(){
```

Comment:
http://stackoverflow.com/questions/5082165/add-event-handler-for-body-onload-by-javascript-within-body-part